context/build-utils/aliases.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { resolve } = require('path');
+
+const aliases = {
+  'metadata-field-descriptions$': resolve('./ingest-validation-tools/docs/field-descriptions.yaml'),
+  js: resolve(__dirname, '../app/static/js/'),
+  'portal-images': resolve(__dirname, '../app/static/portal-images/'),
+  'shared-styles': resolve(__dirname, '../app/static/js/shared-styles/'),
+  package: resolve(__dirname, '../package.json'),
+};
+
+exports.aliases = aliases;
```

Comment:
```suggestion
const alias = {
  'metadata-field-descriptions$': resolve('./ingest-validation-tools/docs/field-descriptions.yaml'),
  js: resolve(__dirname, '../app/static/js/'),
  'portal-images': resolve(__dirname, '../app/static/portal-images/'),
  'shared-styles': resolve(__dirname, '../app/static/js/shared-styles/'),
  package: resolve(__dirname, '../package.json'),
};

exports.alias = alias;
```
and corresponding changes in other files? I agree that the plural makes more sense... but there seems to be a convention, and probably easier just to follow it?

Reply:
Changed, thanks!